.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "genesis"]
 	path = genesis
-	url = ../bsc-genesis-contract.git
+	url = https://github.com/bnb-chain/bsc-genesis-contract.git
```

Comment:
when people fork node-deploy, for example NathanBSC
here url should be https://github.com/bnb-chain/bsc-genesis-contract.git
                         not https://github.com/NathanBSC/bsc-genesis-contract.git